app/controllers/invitations_controller.rb

```diff
@@ -0,0 +1,58 @@
+class InvitationsController < AuthenticatedController
+  def new
+    @invitation = Invitation.new
+  end
+
+  def create
+    @invitation = Invitation.new(
+      invitee_email: invitation_params[:invitee_email],
+      organisation: organisation,
+      user: current_user,
+    )
+
+    if @invitation.save
+      redirect_to organisation_path(organisation), notice: "Invitation sent to #{@invitation.invitee_email}"
+    else
+      render :new
+    end
+  end
+
+  def revoke
+    @invitation = Invitation.find_by(id: params[:id], organisation:)
+  end
+
+  def destroy
+    @invitation = Invitation.find_by(id: params[:id], organisation:)
+
+    @invitation.revoked!
+
+    redirect_to organisation_path(organisation), notice: "Invitation to #{@invitation.invitee_email} has been revoked."
+  rescue StandardError
+    redirect_to organisation_path(organisation), alert: "There was a problem revoking the invitation to #{@invitation&.invitee_email}."
+  end
+
+  # TODO: implement resend action
```

Comment:
```suggestion
```

Reply:
So the bit that's missing here is actually sending notifications. I need to think about how I achieve this and whether we centralise